README.md

```diff
@@ -33,22 +33,22 @@ We're open for [contributions](./CONTRIBUTING.md) !
 Please find further information regarding Apache Solr and its related projects at the following links:
 
 - http://www.typo3-solr.com/
-- https://www.dkd.de/de/leistungen/enterprise-search/
+- https://www.dkd.de/de/leistungen/enterprise-websites/enterprise-search/
 - https://github.com/TYPO3-Solr
 - http://lucene.apache.org/
 - http://lucene.apache.org/solr/
 - http://tomcat.apache.org
 - http://www.eclipse.org/jetty/
 
-To try out Apache Solr for TYPO3 visit [www.typo3-solr.com](http://www.typo3-solr.com) where we've set up a basic configuration. Another showcase can be found here: [search.dkd.de](http://search.dkd.de/solr)
+To try out Apache Solr for TYPO3 visit [www.typo3-solr.com](http://www.typo3-solr.com) where we've set up a basic configuration. 
 
 ![dkd Internet Service GmbH](./Resources/Public/Images/dkd_logo.png)
 
 ## Documentation and Support
 
 -   **Main Documentation:**
 
-    https://docs.typo3.org/typo3cms/extensions/solr/
+   https://docs.typo3.org/p/apache-solr-for-typo3/solr/10.0/en-us/Index.html
```

Comment:
```suggestion
   https://docs.typo3.org/p/apache-solr-for-typo3/solr/master/en-us/Index.html#
```
Would propose to show master docs on master branch.
By branching release-X branches, we can change the link.